src/index.ts

```diff
@@ -10,11 +10,46 @@ export interface GitHubStrategyOptions {
   clientID: string;
   clientSecret: string;
   callbackURL: string;
-  scope?: string;
+  scope?: GitHubScope[] | string;
   allowSignup?: boolean;
   userAgent?: string;
 }
 
+export type GitHubScope =
```

Comment:
Could you add a link to where this is documented? It would help to remember where to check the list in case they add more scopes.
```suggestion
/*
 * @see link here
 */
export type GitHubScope =
```

Reply:
Yeah my bad I forgot to copy the comment!